### PR TITLE
fixed import error

### PR DESCRIPTION
**Pull request by mdclarke**
@Eric89GXL please review encountered import error on mne-python after updating both repos to current master.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
This will work for people who have updated, but it's best not to force people to update if they don't have to. We also need to update the calls to `compute_raw_data_covariance` to refer to the new function. How about something like this?
```
try:
    from mne import compute_raw_covariance  # up-to-date mne-python
except ImportError:  # oldmne-python
    from mne import compute_raw_data_covariance as compute_raw_covariance
```
And then look for the instance(s) of `compute_raw_data_covariance` and replace them with `compute_raw_covariance`.

**Comment by kambysese** [comment]
@Eric89GXL is there a particular reason why this PR is outstanding?

**Comment by Eric89GXL** [comment]
Yeah, I didn't know there was a follow up commit :) @mdclarke one of the (too) many github quirks is that commits are not notified, so it's standard practice to request re-review once comments are addressed. I should be able to look in a bit.

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 
 from __future__ import print_function
-
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: this was actually intentional to separate `future` imports from standard imports

File: mnefun/_mnefun.py [hunk 2]
```diff
@@ -13,25 +12,28 @@
 import glob
 from collections import Counter
 from time import time
-
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: ...and standard library imports from other-library imports

File: mnefun/_mnefun.py [hunk 3]
```diff
@@ -13,25 +12,28 @@
...
 import numpy as np
 from scipy import io as spio
 import matplotlib.pyplot as plt
 from numpy.testing import assert_allclose
-
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: ...and other library imports from `mne` imports

File: mnefun/_mnefun.py [hunk 4]
```diff
@@ -45,7 +47,6 @@
 from mne.utils import run_subprocess
 from mne.report import Report
 from mne.io.constants import FIFF
-
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: ...and `mne` imports from `mnefun` (local) imports

File: mnefun/_mnefun.py [hunk 5]
```diff
@@ -936,7 +938,7 @@ def fix_eeg_files(p, subjects, structurals=None, dates=None, run_indices=None):
         names = [name for name in raw_names if op.isfile(name)]
         # noinspection PyPep8
         if structurals is not None and structurals[si] is not None and \
-                dates is not None:
+                        dates is not None:
```
Comment (Eric89GXL) [inline mnefun/_mnefun.py]: Did you editor complain about this line? The original should be okay according to PEP8, whereas this should be a violation (not a multiple of 4 indent)

**Comment by Eric89GXL** [comment]
Sorry to nitpick this simple change to death, but it's good to standardize on style issues like this from the get-go. Otherwise LGTM.

**Comment by mdclarke** [comment]
@Eric89GXL Sorry! just started using PyCharm, might have mashed some buttons.. also Kam drinks when he codes.

**Comment by Eric89GXL** [comment]
Looks good now except for the unaddressed comments above about removed newlines.

**Comment by kambysese** [comment]
WTH?! I am sure we put those in. I ran a pep8 inspection on the indentation issue and the fix must have reformated the imports. @mdclarke pycharm editor still needs some tweaks. Sorry about that @Eric89GXL. Will push again tomorrow.

**Comment by mdclarke** [comment]
@Eric89GXL ready for merge.

**Comment by Eric89GXL** [comment]
Thanks @mdclarke

**Pull request merged**